crates/flox-rust-sdk/src/models/environment.rs

```diff
@@ -165,36 +276,129 @@ impl<S: State> Environment<S> {
     /// List all Packages in the environment
     ///
     /// Currently unused, supposed to be a cleaned up version of [`Environment<_>::catalog`]
-    pub async fn packages(
+    async fn packages(
         &self,
         _nix: &NixCommandLine,
         _system: impl AsRef<str>,
     ) -> Result<Vec<FloxPackage>, EnvironmentError2> {
         todo!()
     }
 
+    /// Copy the environment to a sandbox directory given by `path`
+    ///
+    /// Typically within [Flox.temp_dir].
+    /// This implementation tries to stay independent of the [Flox] struct for now.
+    async fn modify_in(
```

Comment:
This implementation doesn't quite make sense if you already have the temporary directory stored in `Original` i.e. you shouldn't need to take another path as an argument if you already have the temp dir stored on `Original`.

Reply:
yes should be just `modify` or `make_temporary`